### PR TITLE
 Update of cardinality to following the mets schema

### LINKED ISSUE #101: `metsHdr/agent/name` is not 0..*, but 1..1
SIP12 `metsHdr/agent/name` is defined as 0..* MAY, but METS defines it as 1..1 (mets.xsd, lines 247-263):

```xml
<xsd:element name="metsHdr" minOccurs="0">
  <xsd:complexType>
    <xsd:sequence>
      <xsd:element name="agent" minOccurs="0" maxOccurs="unbounded">
        <xsd:complexType>
          <xsd:sequence>
            <xsd:element name="name" type="xsd:string">
```

Duplicates #102

### LINKED ISSUE #102: Inconsistency in cardinality of `metsHdr/agent/name`
The cardinality requirements for `metsHdr/agent/name` do not follow a consistent logic in relation to the cardinality of their `metsHdr/agent`. According to mets.xsd, `metsHdr/agent/name` cardinality is 1..1, a "conditional MUST," i.e. if a `metsHdr/agent` exists, it MUST have exactly one `metsHdr/agent/name`.

|`metsHdr/agent`|Cardinality|`metsHdr/agent/name`|Cardinality|
|---|---|---|---|
SIP9 Archival creator agent | 0..1 MAY | SIP12 | 0..* MAY
SIP15 Submitting agent | 1..1 MUST | SIP18 | 1..1 MAY
SIP21 Contact person agent | 0..* MAY | SIP24 | 1..1 MUST
SIP26 Preservation agent | 0..1 MAY | SIP29 | 1..1 MAY

Duplicates #101

### PR DESCRIPTION
**Pull request by karinbredenberg**
Update so the requirement follows the mets schema Closes #101 Closes #102

Observe that this PR is a replacement for PR #104 which had been closed but not merged.

### CODE REVIEW

File: profile/E-ARK-SIP.xml [hunk 1]
```diff
@@ -215,14 +216,14 @@
...
-            <requirement ID="SIP12" REQLEVEL="MAY" EXAMPLES="metsHdrElementExample1 metsHdrAgentExample1">
+            <requirement ID="SIP12" REQLEVEL="MUST" EXAMPLES="metsHdrElementExample1 metsHdrAgentExample1">
                 <description>
                     <head>Archival creator agent name</head>
                     <p xmlns="http://www.w3.org/1999/xhtml">The name of the organisation(s) that originally created the data being transferred.</p>
                     <p xmlns="http://www.w3.org/1999/xhtml">Please note that this might be different from the organisation which has been charged with preparing and sending the SIP to the archives.</p>
                     <dl xmlns="http://www.w3.org/1999/xhtml">
                         <dt>METS XPath</dt><dd>metsHdr/agent/name</dd>
-                        <dt>Cardinality</dt><dd>0..*</dd>
+                        <dt>Cardinality</dt><dd>0..1</dd>
```
Comment (prettybits) [inline profile/E-ARK-SIP.xml:226]: Hope you don't mind me jumping in, this should be `1..1`

Comment (jmaferreira) [reply profile/E-ARK-SIP.xml:226]: @karinbredenberg Karin, @prettybits is right. It should be `1..1`.

```xml
<sequence>
	<element name="name" type="xsd:string">
		<annotation>
			<documentation xml:lang="en">
				The element &lt;name> can be used to record the full name of the document agent.
			</documentation>
		</annotation>
	</element>
</sequence>
```

**Comment by jmaferreira** [comment]
@karinbredenberg Any update on this discussion?

File: profile/E-ARK-SIP.xml (hunk 1 above)
Comment (karinbredenberg) [reply profile/E-ARK-SIP.xml:226]: Sorry I missed that. Fixed and PR updated.

**Pull request merged**